bazel/cc_wrapper.py

```diff
@@ -45,10 +42,24 @@ def main():
   # it in the same test.
   if "-static-libstdc++" in sys.argv[1:] or "-stdlib=libc++" in sys.argv[1:]:
     compiler = envoy_real_cxx
-    argv = []
+  else:
+    compiler = envoy_real_cc
+
+  # Either:
+  # a) remove all occurrences of -lstdc++ (when statically linking against libstdc++),
+  # b) replace all occurrences of -lstdc++ with -lc++ (when linking against libc++).
+  if "-static-libstdc++" in sys.argv[1:] or "-stdlib=libc++" in envoy_cxxflags:
+    # Append CXXFLAGS to all C++ targets (this is mostly for dependencies).
+    if envoy_cxxflags and "-std=c++" in str(sys.argv[1:]):
+      argv = envoy_cxxflags.split(" ")
+    else:
+      argv = []
     for arg in sys.argv[1:]:
       if arg == "-lstdc++":
-        pass
+        if "-stdlib=libc++" in envoy_cxxflags:
+          arg.append("-lc++")
+        else:
```

Comment:
you don't need `else` and pass.

Reply:
I know, I've tried to be explicit that there are 2 cases we're supporting here. It's slightly more readable, IMHO, but I'm happy to remove it if you prefer it one way or another.

Reply:
Prefer removing unless you want to add a comment as well explaining this, one of these should be done IMHO.